sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -119,111 +129,226 @@
      */
     public Mono<Response<AttestationSigner[]>> getAttestationSignersWithResponse(Context context) {
```

Comment:
This doesn't need to public, please make them package-private to prevent end users from calling these directly.

Reply:
?? I thought all the WithResponse APIs needed to be public, is that not true?

If the WithResponse APIs aren't required, I'll happily remove them.

Reply:
async XXXwithResponse api w/o the context need to be public and be ServiceMethod. This one is likely there because your sync client depends on it.

Reply:
Nope, that one is there because it was present in the generated code. I made an assumption that the generated code was correct, but it's becoming increasingly clear that the generated code is NOT a valid model for the API surface.

Reply:
Ahh - in that case you can remove them. Generated code generates a lot of overloads as convenience methods that are traditionally used by SDKs. 

Only mandatory methods are XXX(ServiceArguments) and XXXWithResponse(ServiceArguments) for async clients.

Reply:
That makes a ton of sense. When I'm done, I'm going to be doing a complete cleanup of the generated code to remove those (like the sync client methods - I can't get rid of them yet, but I will :))

Reply:
Ok, suggestions about how to fix the fact that I'm *required* to add a Context parameter to the synchronous methods? Am I supposed to add the parameter and then ignore it?